frontend/src/views/SecretMainPage/components/SecretImportListView/SecretImportItem.tsx

```diff
@@ -137,7 +155,44 @@ export const SecretImportItem = ({
             isReplication={isReplication}
           />
         </div>
-        <div className="flex items-center space-x-2 px-4 py-2">
+        <div className="flex items-center space-x-4 px-4 py-2">
+          {lastReplicated && (
+            <Tooltip
+              position="left"
+              className="max-w-md whitespace-normal break-words"
+              content={
+                <div className="flex max-h-[10rem] flex-col overflow-auto ">
+                  <div className="flex self-start">
+                    <FontAwesomeIcon icon={faCalendarCheck} className="pt-0.5 pr-2 text-sm" />
+                    <div className="text-sm">Last Replication</div>
+                  </div>
+                  <div className="pl-5 text-left text-xs">
+                    {lastReplicated
+                      ? format(new Date(lastReplicated), "yyyy-MM-dd, hh:mm aaa")
+                      : "-"}
+                  </div>
+                  {!isReplicationSuccess && (
+                    <>
+                      <div className="mt-2 flex self-start">
+                        <FontAwesomeIcon icon={faXmark} className="pt-1 pr-2 text-sm" />
+                        <div className="text-sm">Fail reason</div>
+                      </div>
+                      <div className="pl-5 text-left text-xs">{replicationStatus}</div>
+                    </>
+                  )}
+                </div>
+              }
+            >
+              <div
+                className={twMerge(
+                  "opacity-0 group-hover:opacity-100",
+                  !isReplicationSuccess && "text-red-600"
+                )}
+              >
+                <FontAwesomeIcon icon={isReplicationSuccess ? faInfoCircle : faWarning} />
+              </div>
+            </Tooltip>
```

Comment:
what do you think about adding a warning prompt here that would inform the user that if a secret already exists in that env, it will be overwritten if the same secret key exists in the imported path? Not sure what the best way to put it though 
<img width="880" alt="image" src="https://github.com/Infisical/infisical/assets/65645666/b63003ec-fd15-4063-b310-1e568b1aabc4">